setup.py

```diff
@@ -3,6 +3,8 @@
 install_requires = [
     'torch',
     'numpy',
+    'onnx',
```

Comment:
Better to install onnx packages only via the CI config for unit testing, but do not make it a requirement for the package

Reply:
Onnx is now optional and tests that require it will fail gracefully